public/oldIndex.html

```diff
@@ -90,7 +90,7 @@ <h3 class="mb-0">Helpful Tips During Social Distancing</h3>
                         <strong class="d-inline-block mb-2 text-primary">Workshop</strong>
                         <h3 class="mb-0">SMART GOALS</h3>
                         <div class="mb-1 text-muted">March 11</div>
-                        <p class="card-text mb-auto">This workshop provides an overview of a simple but effective model ...</p>
+                        <p class="card-text mb-auto">This workshop provides an overview of a simple but effective model for identifying, planning for business, personal, ...</p>
```

Comment:
were can we add this to view on react app

Reply:
done